odoo/addons/test_apikeys/__manifest__.py

```diff
@@ -2,7 +2,7 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 {
     'name': 'Tests flow of API keys',
-    'category': 'Tools',
+    'category': 'Hidden/Tools',
```

Comment:
```suggestion
    'category': 'Hidden/Tests',
```

All modules prefixed with `test_` are tests modules ;).

Reply:
@updated categories of test modules to Hidden/Tests

Reply:
missed to update the modules in that path, updated the pr